python/llm/dev/benchmark/benchmark_util.py

```diff
@@ -510,9 +510,12 @@ class BenchmarkWrapper:
     learn more about decoding strategies refer to the [text generation strategies guide](../generation_strategies).
     """
     
-    def __init__(self, model, do_print=True):
+    def __init__(self, model, do_print=False):
```

Comment:
shall we leave the default value as true?

Reply:
> shall we leave the default value as true?
From the disscusion in https://github.com/intel-analytics/BigDL/pull/8871, we want to hide the print by default.